src/main/java/net/minecraftforge/fluids/BlockFluidBase.java

```diff
@@ -109,25 +113,38 @@
     }
     protected Map<Block, Boolean> displacements = Maps.newHashMap();
 
+    private static final class UnlistedPropertyBool extends Properties.PropertyAdapter<Boolean>
+    {
+        public UnlistedPropertyBool(String name)
+        {
+            super(PropertyBool.create(name));
+        }
+    }
+
     public static final PropertyInteger LEVEL = PropertyInteger.create("level", 0, 15);
     public static final PropertyFloat[] LEVEL_CORNERS = new PropertyFloat[4];
-    public static final PropertyFloat FLOW_DIRECTION = new PropertyFloat("flow_direction");
-    public static final ImmutableList<IUnlistedProperty<Float>> FLUID_RENDER_PROPS;
+    public static final PropertyFloat FLOW_DIRECTION = new PropertyFloat("flow_direction", -1000f, 1000f);
```

Comment:
What was this limited? Why 1000? What does it fix?

Reply:
This shouldn't fix anything as the values supplied should always be in this range - it's just attempting to more explicitly state the assumed limits on the values made by the rendering code that uses them.